.github/workflows/test-upstream.yml

```diff
@@ -109,6 +120,13 @@ jobs:
           mamba-version: "*"
           channels: conda-forge,nodefaults
           channel-priority: strict
+      - name: Optionally update upstream cargo dependencies
+        if: github.event.schedule == '0 3 * * *'
+        env:
+          UPDATE_ALL_CARGO_DEPS: false
+        run: |
+          cd dask_planner
+          bash update-dependencies.sh
       - name: Install dependencies and nothing else
         run: |
           mamba install setuptools-rust
```

Comment:
I wonder if there's an easy way of conditionally updating the reporting issue title based on which cron run it was. 
cc: @charlesbluca

Reply:
Think we should be able to pass env variable into the `script` being used to generate the issue, so maybe something like

```
      - name: Report failures
        uses: actions/github-script@v3
        env:
          which_upstream: ${{ github.event.schedule == '0 3 * * *' && 'DataFusion' || 'Dask' }}
        with:
          github-token: ${{ secrets.GITHUB_TOKEN }}
          script: |
            const title = "⚠️ Upstream ${{ env.which_upstream }} CI failed ⚠️"
```